modules/@apostrophecms/rich-text-widget/ui/apos/tiptap-extensions/Document.js

```diff
@@ -0,0 +1,24 @@
+// Acts a custom Document extension
```

Comment:
specifying our own schema content gets us close to being able to 'specify the default rich text element' by prioritizing one node type over another.

Reply:
Acts as a (sorry)